src/universal/conf/testmode.conf

```diff
@@ -0,0 +1,24 @@
+mantis {
+
+  testmode = true
+
+  datadir = ${user.home}"/.mantis-testmode"
+
+  sync.do-fast-sync = false
+
+  network.discovery {
+    discovery-enabled = false
+    bootstrap-nodes = []
+  }
+
+  pruning.mode = "archive"
+
+  blockchain = {
+    eip160-block-number = "0"
+  }
+
+  network.rpc {
+    apis = "eth,web3,net,personal,daedalus,test"

```

Comment:
this should include iele if we want to use testmode for the solidity-to-iele test suite, right?